Tests/SonarScanner.MSBuild.PreProcessor.Test/WebClientDownloaderBuilderTest.cs

```diff
@@ -681,6 +765,10 @@ public async Task IntermediateCAsWithDifferentIntermediates_Fail()
         // A trusted chain can not be build, because intermediateCAServer is not send by the server. The intermediateCATrustStore found in the trust store is used to build
         // the chain, but the chain status contains the error "The signature of the certificate cannot be verified.".
         await ShouldThrowServerValidationFailed(downloader);
+        logger.AssertDebugLogged($"""
+            The remote server certificate is not trusted by the operating system. The scanner is checking the certificate against the certificates provided by the file '{trustStoreFile.FileName}' (specified via the sonar.scanner.truststorePath parameter or its default value).
+            """);
+        logger.Warnings.Should().ContainSingle(because: "the warning is either WARN_TrustStore_Chain_Invalid or WARN_TrustStore_OtherChainStatus depending on the environment.");
```

Comment:
The code path taken is different in CI and locally. It doesn't really matter what code path fails at the end, so having at least some warning is good enough).

See this CI failure for the two warnings that are issued locally and in CI:
https://dev.azure.com/sonarsource/DotNetTeam%20Project/_build/results?buildId=110200&view=logs&j=3635ba3c-75de-526b-7d9e-c1be443d4750&t=8957dee2-f992-5acc-d4ff-d7b1b1efb951&l=193